sdk/boms/azure-sdk-bom/README.md

```diff
@@ -74,3 +74,24 @@ Jackson, Netty, OkHttp, SLF4J, and more.
   </dependency>
 </dependencies>
 ```
+
+### Overview
+
+#### Release cadence
+
+Currently, the `azure-sdk-bom` releases on a quarterly cycle, excluding the end of year quarter (November - January).
+Releases will occur in February, May, and August using the latest GA versions of managed libraries which meet the
+requirement for release.
+
+#### Requirements for release
+
+The `azure-sdk-bom` has additional release requirements not found in other Azure SDKs. For the `azure-sdk-bom` to be
+considered as a release candidate all managed Azure SDK libraries contained within in it must have strong alignment
+on the Core SDK libraries. For example, all libraries must be using the same version of `azure-core` or to be a viable
+release candidate.
+
+#### Included library requirements
+
+Only GA'd Azure SDKs are permitted to be managed dependencies in the `azure-sdk-bom`. SDKs that have yet to be GA'd
```

Comment:
We should also make a note that this bom only includes managed libraries in `com.azure` group only and no packages from `com.microsoft.azure` are included.